src/main/services/transactions/transaction_service.ts

```diff
@@ -0,0 +1,96 @@
+import { Resource } from "@companieshouse/api-sdk-node";
+import { Transaction } from "@companieshouse/api-sdk-node/dist/services/transaction/types";
+import logger from "../../../../lib/Logger";
+import { createPublicOAuthApiClient } from "../api/api_service";
+import { Session } from "@companieshouse/node-session-handler";
+import ApiClient from "@companieshouse/api-sdk-node/dist/client";
+import { ApiErrorResponse, ApiResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
+import { StatusCodes } from "http-status-codes";
+
+import { CREATE_DESCRIPTION, REFERENCE, transactionStatuses } from "../../config";
+
+/**
+ * Post transaction
+ */
+export const postTransaction = async (session: Session, companyNumber: string, description: string, reference: string): Promise<Transaction> => {
```

Comment:
Aren't we suppose to use the acsp-api for create trasaction?

Reply:
I am being told that Transaction can only be created from web layer. I dont quite agree with that.. but I will tackle that when I will work on transaction model.